lib/process-release.js

```diff
@@ -1,11 +1,11 @@
-/* eslint-disable node/no-deprecated-api */
+/* eslint-disable n/no-deprecated-api */
```

Comment:
guess this line is not needed if this didn't cause issues?

Reply:
The update of `standard` to the latest version uses a new eslint plugin called `eslint-plugin-n` (https://www.npmjs.com/package/eslint-plugin-n). So disabling this rule had to change to the plugin name.